docs/src/starknet/101.md

```diff
@@ -0,0 +1,209 @@
+# `sncast` 101
+
+This page showcases the standard usage of various `sncast` commands.
+For more in-depth explanation, please follow other guides from "sncast Overview".
+
+## Create and Deploy an Account
+
+First, create an account.
+This generates the account details but doesn't put it on the network yet.
+
+<!-- { "ignored": true } -->
+```shell
+$ sncast account create \
+  --name my_account \
+  --network sepolia
```

Comment:
nit: We can even reduce it to be `sncast account create --network sepolia`, the name will be auto-generated (that's up to you).

Reply:
I think we can show picking a specific name, it makes more sense for the following commands.